spirv_msl.cpp

```diff
@@ -7144,6 +7320,7 @@ void CompilerMSL::emit_instruction(const Instruction &instruction)
 		}
 		else
 			CompilerGLSL::emit_instruction(instruction);
+		emit_interpolant_access_chain(ops, instruction.length);
```

Comment:
Sounds like this should be called fixup interpolant access chain or something like that.